postgresql-11.5/src/fe_utils/printbteq.c

```diff
@@ -573,6 +573,33 @@ _print_horizontal_line(const unsigned int ncolumns, const unsigned int *widths,
     fputc('\n', fout);
 }
 
+static void
```

Comment:
Please, add comments to the function to explain what this does.

Reply:
done as specified